distribution/benchmark/bproducer.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh

```

Comment:
Do we have any descriptions in one place? if not, I suggest you could rename it to batchproducer.sh

Reply:
ok